recipes/libzip/all/conanfile.py

```diff
@@ -19,15 +21,17 @@ class LibZipConan(ConanFile):
         "with_bzip2": [True, False],
         "with_lzma": [True, False],
         "with_zstd": [True, False],
-        "crypto": [False, "win32", "openssl", "mbedtls", "auto"]
+        "crypto": [False, "win32", "openssl", "mbedtls"],
```

Comment:
might be breaking for consumers?

Reply:
yes, but I don't know how to deprecate an option value.

Reply:
@uilianries I remember you had an example for that

Reply:
Yes, https://github.com/conan-io/conan-center-index/blob/d7facaec13193af24231aba173d3638aa0bec4ae/recipes/dcmtk/all/conanfile.py#L24

I'll add a section to FAQs

Reply:
No, it's different, and we did that the first time in libcurl. I don't want to deprecate an option, but an option value.

Reply:
Oh, actually we already have it documented: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#can-i-remove-an-option-from-recipe

Reply:
can we add something like:
```
if self.options.crypto == "auto":
    self.output.warn("auto was deprecated")
    self.options.crypto = "win32"
```

Reply:
Yes I guess

It would become:

```python
if self.options.crypto == "auto":
    self.output.warn("auto was deprecated")
    self.options.crypto = "win32" if self.settings.os == "Windows" else "openssl"
```